portfolio/frontend/src/containers/Login/Login.js

```diff
@@ -1,14 +1,15 @@
 import React from "react"
 import classes from "./Login.module.css"
+import {Link} from "react-router-dom"
 
 const Login = (props) => {
     return (
         <div>
             <div className={classes.Header}>
                 <img src="sGonksLogo.png" alt="sGonks Logo" className={classes.Logo}></img>
-                <div className={classes.HeaderButtons}>
-                    <button className={classes.Login} onClick={() => props.handler()}>Login</button>
-                    <button className={classes.Help}>See how it works</button>
+                <div className={classes.Buttons}>
+                    <Link className={classes.Login} to="/sgonks-platform">Login</Link>
+                    <Link className={classes.Help} to="/explanation">See how it works</Link>
                 </div>
```

Comment:
Can this be factored out with the buttons on line 24 and 25

Reply:
I'm not sure what you mean. Could you clarify?

Reply:
Factor out common code to function

Reply:
Can you do that with html?